templates/commands/mtg_card.html

```diff
@@ -0,0 +1,7 @@
+<a href="{{ scryfall_uri }}">
+  <div style="float: left">
+    {% for img_uri in img_uris %}
+      <img src="{{ img_uri }}" width="251", height="350" style="border-radius: 4.75% / 3.5%; margin-right: 5px" />
```

Comment:
width e height sono come attributi e non css perché se non li metti così showdown si lamenta? comunque c'è una virgola in più dopo il width="251"
e niente / alla fine pls che non mi piace

Reply:
Sì, PS richiede che esistano i 2 attributi, ma se li metti all'interno di `style` non vengono riconosciuti. Grazie per la virgola però.